### PR TITLE
Add test for Breaking change DroppedSupportOfServiceLoaderCondition

### PR DESCRIPTION
**Pull request by Seiphon**

### CODE REVIEW

**Comment by Seiphon** [comment]
ci:test

**Comment by jtydhr88** [comment]
SF Successfully on liferay-ide folder

File: tools/tests/com.liferay.ide.upgrade.problems.core.tests/projects/filetests/DroppedSupportOfServiceLoaderConditionTest.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.test;
+
+import com.liferay.portal.kernel.util.DefaultServiceLoaderCondition;
+import com.liferay.portal.kernel.util.ServiceLoaderCondition;
+
+public class DroppedSupportOfServiceLoaderConditionTest {
```
Comment (jtydhr88) [inline tools/tests/com.liferay.ide.upgrade.problems.core.tests/projects/filetests/DroppedSupportOfServiceLoaderConditionTest.java]: can you copy the real code from portal for testing?

**Comment by jtydhr88** [comment]
see comments and resend, thanks.

**Comment by Seiphon** [comment]
> see comments and resend, thanks.

I didn't find the real code from portal code, there is no real invocation for this method of the Class.

**Pull request closed without merging**